safe_transaction_service/history/tasks.py

```diff
@@ -472,3 +479,58 @@ def send_webhook_task(address: Optional[str], payload: Dict[str, Any]) -> int:
 
         sent_requests += 1
     return sent_requests
+
+
+@app.shared_task(
+    bind=True,
+    soft_time_limit=SOFT_TIMEOUT,
+    autoretry_for=(MetadataRetrievalExceptionTimeout,),
```

Comment:
First time this retry task is executed by an explicit call to `retry_get_metadata_task` and then it auto-retried every time `MetadataRetrievalExceptionTimeout` is thrown, right?

Reply:
yes, the first retry task is going to be executed after countdown, and then auto-retried following `exponential backoff` delay between retries, here more context, https://docs.celeryq.dev/en/stable/userguide/tasks.html#automatic-retry-for-known-exceptions. 
Reading again the documentation maybe is better instead to define a `maximum number of retries`, define a `maximum number of seconds`. What do you think? I'm worried about default value that is less than the time for the fourth retry.

Reply:
Hmm, according the docs, setting `retry_backoff` to `3`:
```
For example, if this option is set to 3, the first retry will delay 3 seconds, the second will delay 6 seconds, the third will delay 12 seconds, the fourth will delay 24 seconds, and so on.
```

So it depends on which time we think it's appropriate to be retrying. Maybe it's more readable by setting the `retry_backoff_max` to a number of seconds as you said (one hour for example).

Reply:
It's true, then I must modify the 3 for a higher one then.

Reply:
Changed to 60 seconds